distributed/tests/test_client.py

```diff
@@ -625,6 +625,10 @@ async def test_get(c, s, a, b):
 
 def test_get_sync(c):
     assert c.get({"x": (inc, 1)}, "x") == 2
+    try:
+        c.get({"y": (inc, 1)}, "z")
+    except Exception as e:
+        assert "not found in the dask graph" in str(e)
```

Comment:
As @sublinus pointed out, we tend to use `pytest.raises` to check that an error has been raised. This provides useful feedback when the expected error isn't raised and has nice error message checking functionality. In general we also try to be as specific as we can when catching errors. Here that would mean catching a `KeyError` instead of `Exception`. Something like the following would be the  `pytest.raises` equivalent: 

```python
with pytest.raises(KeyError, match="not found in the dask graph"):
    c.get({"y": (inc, 1)}, "z")
```